src/scss/components/ec-btn/_ec-btn.scss

```diff
@@ -0,0 +1,153 @@
+// Here we state the default value of each color, if we change a brand
+// we can override the default value with something else
+$ec-btn-color: $white !default;
+$ec-btn-primary-color: $color-info !default;
```

Comment:
this is wrong.
the primary-color is _$level-4-tech-blue_
like that 
![image](https://user-images.githubusercontent.com/29793161/68392016-c44b9f80-0160-11ea-91a1-6b5dbb06deef.png)